package.json

```diff
@@ -83,7 +83,7 @@
     "resin.io",
     "git"
   ],
-  "author": "Juan Cruz Viotti <juan@balena.io>",
+  "author": "Balena Inc. (https://balena.io/)",
```

Comment:
* The `package.json` documentation (npm v6 and v7) states that the `author` field is an abbreviated version of the `name`, `email` and `url` fields, where `email` and `url` are optional, in the format:
  * `"author": "Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"`
  * https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors

  I considered using the `name` and `url` fields separately instead of `author`, however `oclif` (and perhaps other tools) look for the `author` field specifically:
  * https://github.com/oclif/dev-cli/blob/v1.26.0/src/commands/pack/win.ts#L46
  * https://github.com/oclif/dev-cli/blob/v1.26.0/src/commands/pack/win.ts#L66-L67

  Given that the email address is optional, I chose to use the URL instead of an email address. The website provides further contact details, and so does `balena help`.

* Etcher's `package.json` currently shows:
  * `"author": "Balena Inc. <hello@etcher.io>"`
  * https://github.com/balena-io/etcher/blob/v1.5.120/package.json#L46

  This seems to come from a commit of 13 June 2017 that worded it Resin Inc. to match the Windows signing certificate at the time: https://github.com/balena-io/etcher/blame/9cc65a386bf6887df5b29261533464dccb0f0017/package.json
  The Windows signing certificate current reads Balena Inc., which is a USA company. Interestingly, the singing certificate for macOS reads Balena Ltd., and so does the licensing excerpts at the top of source code files in the CLI's codebase. Balena Ltd. is a UK company and if I remember correctly, Balena Ltd. owns Balena Inc. I don't think it matters much whether we use Inc or Ltd here, and I chose Inc so that Windows Program and Features list would look nicer with both Etcher and balena CLI showing the same company name.